.github/workflows/ansible-test.yml

```diff
@@ -17,6 +17,9 @@ jobs:
         with:
           python-version: 3.6
 
+      - name: Install ansible.netcommon
```

Comment:
Needs to be after you install ansible

Reply:
From the logs, we can see this is currently being installed to `Installing 'ansible.netcommon:1.0.0' to '/home/runner/.ansible/collections/ansible_collections/ansible/netcommon`

Add ` -p ../../`